src/lib/base58_check/version_bytes.ml

```diff
@@ -4,48 +4,54 @@ type t = char
 
 (* each of the following values should be distinct *)
```

Comment:
I'm all for having these be a little less random, but might this version byte change force us to do the whole "generate a new public key" thing next release?

Reply:
Hm, yes, and it's not the private_key byte that needs to stay the same, the non_zero_curve_points are the actually-important ones.

Reply:
I think I've preserved the important values now.